toolkit/docs/building/prerequisites-ubuntu.md

```diff
@@ -18,7 +16,7 @@ sudo apt -y install \
     gawk \
     genisoimage \
     git \
-    golang-1.19-go \
+    golang-1.20-go \
```

Comment:
Update the go version in the `go.mod` file as well?

Reply:
go.mod needs to be updated as well, because we started using the 1.20 version in the toolkit, starting this [PR](https://github.com/microsoft/CBL-Mariner/pull/6336)

go tools testing fails for imagecustomizerlib with go version 1.19 on Ubuntu.
This is not caught in the build pipelines because build pipelines are already at 1.20.7

Reply:
Huh, I completely missed the fact, that we were on 1.19, not 1.20 in `go.mod`. Sure, I can update `go.mod` as well.